physical/mysql.go

```diff
@@ -1,14 +1,22 @@
 package physical
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"database/sql"
 	"fmt"
+	"io/ioutil"
+	"net/url"
 	"sort"
 	"strings"
 	"time"
 
 	"github.com/armon/go-metrics"
-	_ "github.com/go-sql-driver/mysql"
+	mysql "github.com/go-sql-driver/mysql"
+)
+
+var (
+	tlsKey = "custom"
```

Comment:
Can you add a commend to what this is? Also it is better as a `const` with a prefix like `mysqlTLSKey` since otherwise it could collide in the `physical` package